Helm/opensearch-dashboards/templates/service.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "opensearch-dashboards.fullname" . }}
+  labels: {{ include "opensearch-dashboards.labels" . | nindent 4 }}
```

Comment:
This should use `{{- ... }}` to avoid a trailing space on the line after `label: `.

Reply:
Changes made in the PR